chainer/initializers/constant.py

```diff
@@ -17,8 +17,9 @@ class Identity(initializer.Initializer):
 
     """
 
-    def __init__(self, scale=1.0):
+    def __init__(self, scale=1.0, **kwargs):
```

Comment:
I think it is better to explicitly write down the dtype argument and add it to the docstring.